packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -42,23 +44,30 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
             textAlign: "center",
             width: "100%",
             zIndex: "100",
+            "& > :first-child:nth-last-child(1)": {
+                display: "none",
+            },
         },
         carousel_sequenceIndicator: {
             display: "inline-block",
-            background: "transparent",
-            border: "0",
-            height: "10px",
-            padding: "2px 3px",
-            width: "10px",
+            padding: "0 2px",
             "&:focus": {
                 outline: "none",
             },
             "&::before": {
-                borderRadius: "50%",
+                opacity: "0.2",
+                border: "1px solid transparent",
+                borderRadius: "40px",
                 content: "''",
                 display: "block",
-                height: "100%",
-                width: "100%",
+                height: "4px",
+                width: "32px",
+                transition: "all 0.2s ease-in-out",
```

Comment:
just checking, but are none of these values based on the design system?

Reply:
See other response.